ivy/functional/ivy/elementwise.py

```diff
@@ -4232,6 +4232,39 @@ def log2(
     Both the description and the type hints above assumes an array input for simplicity,
     but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
     instances in place of any of the arguments.
+
+    Examples
+    --------
+    With :class:`ivy.Array` input:
+    >>> x = ivy.array([5.0, 1, -0.0, -6.0])
+    >>> y = ivy.log2(x)
+    >>> print(y)
+    ivy.array([2.32, 0., -inf, nan])
+    >>> x = ivy.array([[float('nan'), 1, 6.0, float('+inf')],
+    ...               [+0, -2.0, -7, float('-inf')]])
+    >>> y = ivy.log2(x)
+    >>> print(y)
+    ivy.array([[nan, 0., 2.58, inf],[inf, nan, nan, nan]])
+
+
+    With :class:`ivy.NativeArray` input:
```

Comment:
I would say this is not needed, `ivy.Array` is enough, please delete these examples. In the [deep dive docstrings](https://unify.ai/docs/ivy/overview/deep_dive/docstring_examples.html) examples there is a relevant example, it states:

> Point 1 is simple to satisfy. Ignoring the union over ivy.Array and ivy.NativeArray which is covered by points 6 and 7, and ignoring the nestable nature of the function which is covered by points 8 and 9, then as far as point 1 is concerned, the input x only has one possible variation. It must be an array.